exercises/Version-control-basic-importing-data-R.md

```diff
@@ -9,8 +9,8 @@ This is a follow up to
 [First Commit]({{ site.baseurl }}/exercises/Version-control-basic-first-commit-R).
 
 1. Download a copy of the
-   [main data file]({{ site.baseurl }}/data/houseelf-earlength-dna-data.csv) and
-   save it to the a `data` subdirectory in your project folder.
+   [main data file](https://lter.limnology.wisc.edu/file/11003/download?token=JN-QrebL5udjuX1AQvUt3hUYtzrLW0StCFbgxE5PpGA) and
```

Comment:
This url doesn't work (drops me to a login page), but https://lter.limnology.wisc.edu/sites/default/files/Gaeta_etal_CLC_data.csv seems to work OK.